src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -165,12 +165,8 @@ protected final void initOutput(BatchSinkContext context, BigQuery bigQuery, Str
     List<String> fieldNames = fields.stream()
       .map(BigQueryTableFieldSchema::getName)
       .collect(Collectors.toList());
-    // Get the dataset location (if it exists)
-    DatasetId datasetId = DatasetId.of(getConfig().getDatasetProject(), getConfig().getDataset());
-    Dataset dataset = bigQuery.getDataset(datasetId);
-    String location = dataset != null ? dataset.getLocation() : getConfig().getLocation();
```

Comment:
Removed these lineage because we do not need to send location information anymore